src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -48,6 +48,67 @@ describe('DictionaryOverview', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should handle HEAD version release', () => {
+    const props = {
+      dictionary: [dictionary],
+      versions: [customVersion],
+      dictionaryConcepts: [dictionary],
+      isReleased: false,
+      match: {
+        params: {
+          ownerType: 'testing',
+          owner: 'tester',
+          type: 'collection',
+          name: 'chris',
+        },
+      },
+      fetchDictionary: jest.fn(),
+      fetchVersions: jest.fn(),
+      fetchDictionaryConcepts: jest.fn(),
+      releaseHead: jest.fn(),
+    };
+    const wrapper = mount(<Provider store={store}><MemoryRouter><DictionaryOverview {...props} /></MemoryRouter></Provider>);
+    expect(wrapper.find('.fas.fa-cloud-upload-alt')).toHaveLength(1);
+    expect(wrapper).toMatchSnapshot();
+    wrapper.find('.fas.fa-cloud-upload-alt').simulate('click');
```

Comment:
Great job. However, I think this line that simulates a click should come before the two assertions above it.

Reply:
Please review this test to simulate and test what happens when a user clicks what you are clicking.

Reply:
Thanks for noticing this. Will work on it.